packages/airnode-node/src/config/types.ts

```diff
@@ -1,124 +1 @@
-  readonly templateId: string;
-  readonly endpointId: string;
-  readonly encodedParameters: string;
-}
-
-// ===========================================
-// Logging
-// ===========================================
-export type LogLevel = 'DEBUG' | 'INFO' | 'WARN' | 'ERROR';
-
-export type LogFormat = 'json' | 'plain';
-
-export type ChainType = 'evm'; // Add other blockchain types here;
-
-export interface ChainContracts {
-  readonly AirnodeRrp: string;
-}
-
-export interface Provider {
-  readonly url: string;
-}
-
-export interface PriorityFee {
-  readonly value: number;
-  readonly unit?: 'wei' | 'kwei' | 'mwei' | 'gwei' | 'szabo' | 'finney' | 'ether';
-}
-
-export interface ChainOptions {
-  readonly txType: 'legacy' | 'eip1559';
-  readonly gasPriceMultiplier?: number;
-  readonly baseFeeMultiplier?: number;
-  readonly priorityFee?: PriorityFee;
-  readonly fulfillmentGasLimit: number;
-}
-
-export interface ChainConfig {
-  readonly authorizers: string[];
-  readonly blockHistoryLimit?: number;
-  readonly contracts: ChainContracts;
-  readonly id: string;
-  readonly minConfirmations?: number;
-  readonly type: ChainType;
-  readonly options: ChainOptions;
-  readonly providers: Record<string, Provider>;
-  readonly maxConcurrency: number;
-}
-
-export interface HttpGateway {
-  readonly enabled: boolean;
-  readonly apiKey?: string;
-  readonly maxConcurrency?: number;
-}
-
-export interface Heartbeat {
-  readonly enabled: boolean;
-  readonly apiKey?: string;
-  readonly id?: string;
-  readonly url?: string;
-}
-
-export interface LocalProvider {
-  readonly type: 'local';
-}
-
-export interface AwsCloudProvider {
-  readonly type: 'aws';
-  readonly region: string;
-  readonly disableConcurrencyReservations: boolean;
-}
-
-export interface GcpCloudProvider {
-  readonly type: 'gcp';
-  readonly region: string;
-  readonly projectId: string;
-  readonly disableConcurrencyReservations: boolean;
-}
-
-export type CloudProvider = AwsCloudProvider | GcpCloudProvider;
-export type LocalOrCloudProvider = LocalProvider | CloudProvider;
-
-export interface NodeSettings {
-  readonly airnodeWalletMnemonic: string;
-  readonly heartbeat: Heartbeat;
-  readonly httpGateway: HttpGateway;
-  readonly httpSignedDataGateway: HttpGateway;
-  readonly airnodeAddressShort?: string;
-  readonly stage: string;
-  readonly cloudProvider: LocalOrCloudProvider;
-  readonly logFormat: LogFormat;
-  readonly logLevel: LogLevel;
-  readonly nodeVersion: string;
-}
-
-export interface ApiCredentials extends BaseApiCredentials {
-  readonly oisTitle: string;
-}
-
-// TODO: Test that the parse result from validator is compatible with this config
-export interface Config {
-  readonly chains: ChainConfig[];
-  readonly nodeSettings: NodeSettings;
-  readonly ois: OIS[];
-  readonly triggers: Triggers;
-  readonly templates: Template[];
-  readonly apiCredentials: ApiCredentials[];
-}
+export * from '@api3/airnode-validator/dist/src/config';
```

Comment:
In the end, I've opted for this approach.

Doing `export { config } from '@api3/airnode-validator'` makes cumbersome imports and re-exporting everything one by one is tedious and needs to be updated every time we want to add something new. Let me know what you think or whether this approach might break something.

Reply:
We package different targets of the validator. This only works, because of a bug that I've introduced when I've added the references (read details below). Anyway, what you should use is `@api3/airnode-validator/dist/cjs/src/config`.

While testing this I realized that the `dist` folder is shared for tsconfig.build.json and tsconfig.json, but they build different files. This is essentially https://api3dao.atlassian.net/browse/AN-762 and the fix is probably to reference the tsconfig.build.json files instead.

Reply:
Btw. can we remove this file and just use the `export * ...` in `index.ts` ? Since the `types.ts` file exports also the schemas from the validator/ois. Having the file named `types.ts` it is mislieading.

Reply:
> While testing this I realized that the `dist` folder is shared for tsconfig.build.json and tsconfig.json, but they build different files. This is essentially https://api3dao.atlassian.net/browse/AN-762 and the fix is probably to reference the tsconfig.build.json files instead.

Or, to get rid of `tsconfig.build.json` as you did on Airseeker. I would actually prefer that.